interface-definitions/service_ids_suricata.xml.in

```diff
@@ -0,0 +1,217 @@
+                      <valueHelp>
+                        <format>!ipv6</format>
+                        <description>Exclude the specified IPv6 address from matches</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>!ipv4net</format>
+                        <description>Exclude the specified IPv6 prefix from matches</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>!ipv6net</format>
+                        <description>Exclude the specified IPv6 prefix from matches</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ipv4-address"/>
+                        <validator name="ipv6-address"/>
+                        <validator name="ipv4-prefix"/>
+                        <validator name="ipv6-prefix"/>
+                        <validator name="ipv4-address-exclude"/>
+                        <validator name="ipv6-address-exclude"/>
+                        <validator name="ipv4-prefix-exclude"/>
+                        <validator name="ipv6-prefix-exclude"/>
+                      </constraint>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="group">
+                    <properties>
+                      <help>Address group</help>
+                      <completionHelp>
+                        <path>service ids suricata address-group</path>
+                        <list>home-net external-net http-servers smtp-servers sql-servers dns-servers telnet-servers aim-servers dc-servers dnp3-server dnp3-client modbus-client modbus-server enip-client enip-server</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>string</format>
+                        <description>Address group to match</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>!string</format>
+                        <description>Exclude the specified address group from matches</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>!?[a-z0-9-]+</regex>
+                      </constraint>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                </children>
+              </tagNode>
+              <tagNode name="port-group">
+                <properties>
+                  <help>Port group name</help>
+                  <completionHelp>
+                    <list>http-ports shellcode-ports oracle-ports ssh-ports dnp3-ports modbus-ports file-data-ports ftp-ports geneve-ports vxlan-ports teredo-ports</list>
+                  </completionHelp>
+                  <constraint>
+                    <regex>[a-z0-9-]+</regex>
```

Comment:
missing valueHelp and a too relaxed regex. Should be:

`<regex>(http-ports|shellcode-ports|oracle-ports|ssh-ports|dnp3-ports|modbus-ports|file-data-ports|ftp-ports|geneve-ports|vxlan-ports|teredo-ports)</regex>`

Reply:
The relaxed `port-group` values is due to the same reasoning as the above `address-group` values.